TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -28,6 +29,38 @@ public class TypeCobolOptions : IBMCompilerOptions
         /// Shall we use Antlr for Parsing the Program.
         /// </summary>
         public bool UseAntlrProgramParsing { get; set; }
+        /// <summary>
+        /// Shall we use EUROINFO_LEGACY_REPLACING_SYNTAX snippets for Parsing the Program.
+        /// </summary>
+        public bool UseEuroInformationLegacyReplacingSyntax { get; set; } = true;
+
+#if EUROINFO_RULES
+#else
+        private bool _useEuroInformationLegacyReplacingSyntax;
```

Comment:
I think you meant UseEuroInformationLegacyReplacingSyntax = false ?

Reply:
It's a refactoring mistake. I fixed it in the latest push